src/shared/ui/Counter/Counter.jsx

```diff
@@ -0,0 +1,29 @@
+import { Component, createElement } from "react";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { number: props.initialValue || 0  };
```

Comment:
it's better to use new syntax `??`, we use it when  the value can be missed and we need a default value 

also, we need to use `?` as well because we're not sure we get an initial value 

```suggestion
    this.state = { number: props?.initialValue ?? 0  };
```

Reply:
okay done